packages/oasis3-mct/package.py

```diff
@@ -18,6 +18,7 @@ class Oasis3Mct(MakefilePackage):
     maintainers = ["harshula"]
 
     version("master", branch="master")
+    version("access-esm1.5", branch="access-esm1.5")
```

Comment:
I was going to ask if the `access-esm1.5` branch is protected, but I see none of them are. Have made an issue

https://github.com/ACCESS-NRI/oasis3-mct/issues/22

This is a comment, it does not require any changes.